### ISSUE TITLE
Tests for French lemmatization are skipped (but the model is installed)

### ISSUE THREAD

**Issue by amperinet** [explosion/spaCy#2991]
## How to reproduce the problem
I modified the lemmatization files for French (see PR #2992) and went through the tests that are in /spacy/tests/lang/fr, but the tests for lemmatization are skipped and I don't understand why.
From what I read on the tests git page, if a test is skipped it is because the model is not found. So I did as many checks as possible related to the models. 

I ran "pytest -v ." from the French tests directory ('spacy/tests/lang/fr/').

I tested on both Linux and Mac. 

### 1 - Mac 

```bash
MacBook-Pro-2:fr amperinet$ pytest -v .
========================================================================= test session starts =========================================================================
platform darwin -- Python 3.6.5, pytest-4.0.1, py-1.7.0, pluggy-0.8.0 -- /Library/Frameworks/Python.framework/Versions/3.6/bin/python3.6
cachedir: .pytest_cache
rootdir: /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/spacy/tests/lang/fr, inifile:
collected 18 items                                                                                                                                                    

test_exceptions.py::test_tokenizer_infix_exceptions[aujourd'hui] PASSED                                                                                         [  5%]
test_exceptions.py::test_tokenizer_infix_exceptions[Aujourd'hui] PASSED                                                                                         [ 11%]
test_exceptions.py::test_tokenizer_infix_exceptions[prud'hommes] PASSED                                                                                         [ 16%]
test_exceptions.py::test_tokenizer_infix_exceptions[prud\u2019hommal] PASSED                                                                                    [ 22%]
test_exceptions.py::test_tokenizer_handles_abbr[janv.-janvier] PASSED                                                                                           [ 27%]
test_exceptions.py::test_tokenizer_handles_abbr[juill.-juillet] PASSED                                                                                          [ 33%]
test_exceptions.py::test_tokenizer_handles_abbr[Dr.-docteur] PASSED                                                                                             [ 38%]
test_exceptions.py::test_tokenizer_handles_abbr[av.-avant] PASSED                                                                                               [ 44%]
test_exceptions.py::test_tokenizer_handles_abbr[sept.-septembre] PASSED                                                                                         [ 50%]
test_exceptions.py::test_tokenizer_handles_exc_in_text PASSED                                                                                                   [ 55%]
test_exceptions.py::test_tokenizer_handles_exc_in_text_2 PASSED                                                                                                 [ 61%]
test_exceptions.py::test_tokenizer_handles_title PASSED                                                                                                         [ 66%]
test_exceptions.py::test_tokenizer_handles_title_2 PASSED                                                                                                       [ 72%]
test_lemmatization.py::test_lemmatizer_verb[fr_core_news_sm] SKIPPED                                                                                            [ 77%]
test_lemmatization.py::test_lemmatizer_noun_verb_2[fr_core_news_sm] SKIPPED                                                                                     [ 83%]
test_lemmatization.py::test_lemmatizer_noun[fr_core_news_sm] SKIPPED                                                                                            [ 88%]
test_lemmatization.py::test_lemmatizer_noun_2[fr_core_news_sm] SKIPPED                                                                                          [ 94%]
test_text.py::test_tokenizer_handles_long_text PASSED                                                                                                           [100%]

================================================================ 14 passed, 4 skipped in 11.36 seconds ================================================================
```

I checked the model for French, but it is well installed. I downloaded again, to be sure I have the right model. 
```
MacBook-Pro-2:site-packages amperinet$ pwd
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages
MacBook-Pro-2:site-packages amperinet$ ll fr_core_news_sm*
fr_core_news_sm:
total 16
-rw-r--r--  1 amperinet  admin   291  7 nov  2017 __init__.py
drwxr-xr-x  3 amperinet  admin    96 28 nov 15:45 __pycache__
drwxr-xr-x  8 amperinet  admin   256 28 nov 15:45 fr_core_news_sm-2.0.0
-rw-r--r--  1 amperinet  admin  1176 28 nov 15:45 meta.json

fr_core_news_sm-2.0.0-py3.6.egg-info:
total 56
-rw-r--r--  1 amperinet  admin   453 28 nov 15:45 PKG-INFO
-rw-r--r--  1 amperinet  admin  1323 28 nov 15:45 SOURCES.txt
-rw-r--r--  1 amperinet  admin     1 28 nov 15:45 dependency_links.txt
-rw-r--r--  1 amperinet  admin  1251 28 nov 15:45 installed-files.txt
-rw-r--r--  1 amperinet  admin     1  7 nov  2017 not-zip-safe
-rw-r--r--  1 amperinet  admin    16 28 nov 15:45 requires.txt
-rw-r--r--  1 amperinet  admin    16 28 nov 15:45 top_level.txt
```

I checked in spacy/tests/conftest.py, but I did not change anything and it seems ok too. 
Results of Validate:
```
    Installed models (spaCy v2.0.17)
    /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/spacy

    TYPE        NAME                  MODEL                 VERSION                                   
    package     fr-core-news-sm       fr_core_news_sm       2.0.0    ✔      
    link        fr                    fr_core_news_sm       2.0.0    ✔      
```
I ran pip3 list, "fr-core-news-sm  2.0.0" shows up. 

### 2 - Linux
I also tested on Linux with an older version of Spacy but I have the same problem, tests for lemmatization are skipped. 
Validate info: 
```
    Installed models (spaCy v2.0.11)
    /usr/local/lib/python3.4/dist-packages/spacy

    TYPE        NAME                  MODEL                 VERSION                                   
    package     en-core-web-sm        en_core_web_sm        2.0.0    ✔      
    package     fr-core-news-sm       fr_core_news_sm       2.0.0    ✔      
    package     fr-core-news-md       fr_core_news_md       2.0.0    ✔      
    package     xx-ent-wiki-sm        xx_ent_wiki_sm        2.0.0    ✔      
    package     fr-depvec-web-lg      fr_depvec_web_lg      1.0.0    --> n/a             
    package     en-core-web-lg        en_core_web_lg        2.0.0    ✔      
    link        xx                    xx_ent_wiki_sm        2.0.0    ✔      
    link        fr                    fr_core_news_sm       2.0.0    ✔      
    link        en                    en_core_web_sm        2.0.0    ✔      
    link        en_core_web_lg        en_core_web_lg        2.0.0    ✔      
    link        fr_core_news_sm       fr_core_news_sm       2.0.0    ✔      
    link        fr_core_news_md       fr_core_news_md       2.0.0    ✔      
```

## Your Environment

Tested with 2 configurations: 
### 1 - Mac
* **spaCy version:** 2.0.17
* **Platform:** Darwin-17.7.0-x86_64-i386-64bit
* **Python version:** 3.6.5
* **Models:** fr

### 2 - Linux
* **Python version:** 3.4.2
* **Platform:** Linux-3.16.0-4-amd64-x86_64-with-debian-8.10
* **spaCy version:** 2.0.11
* **Models:** xx, fr_core_news_sm, en, fr_core_news_md, fr, en_core_web_lg

**Comment by ines** [comment]
Thanks for the report! Maybe this is related to the upgrade we did to pytest 4.x? 🤔 I'll look into this!

In any case, for the upcoming `v2.1.0`, I've refactored the test suite to run much faster and moved the models out to a separate module in our `spacy-models` repo:

https://github.com/explosion/spacy-models/tree/master/tests

Having the model tests included with the library was always dissatisfying, because we weren't really able to run them as part of our regular CI anyways. Instead, we now run the library tests with the library, and the model tests separately as part of our model training pipeline. Having a separate repo also hopefully makes it easier for users to test their own models in a standalone process – for example, the new model tests also include a variety of ["sanity checks"](https://github.com/explosion/spacy-models/blob/master/tests/test_common.py) that every model should pass
